src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -96,18 +97,33 @@ export const DropdownMenu: React.FC<Props> = ({
     }
   };
 
+  /**
+   * Pass props down to children
+   * 1) Cycle through children and pass down ref
+   * 2) If fragment is used for children (ProjectCardDropdown)
+   */
   const childrenWithProps = React.Children.map(
     children,
     // TODO: improve `any` type
-    (child: any, i: number) => {
+    (child: ReactNode, i: number) => {
       // Checking isValidElement is the safe way and avoids a typescript
       // error too.
       if (React.isValidElement(child)) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-expect-error TODO: fix "No overload matches this call" error
-        return React.cloneElement<Props>(child, {
-          ref: (el: HTMLLIElement) => (childRefs.current[i] = el),
-        });
+        if (isReactFragment(child)) {
+          const newChildren = child.props.children;
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          return newChildren.map((item: ReactNode, i: number) => {
+            // @ts-expect-error TODO: fix "No overload matches this call" error
```

Comment:
Is this captured in a ticket somewhere? Wanna make sure we don't lose track of it because it sounds kinda important.

Reply:
That is a good question 😉

Reply:
I don't see one @dierat

Reply:
There is now! https://app.shortcut.com/czi-edu/story/201580/fix-no-overload-matches-this-call-error-in-dropdownmenu-tsx